packages/amazon/src/serverGroup/configure/wizard/instanceType/advancedMode/nonCustom/SortableRow.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Checkbox } from 'react-bootstrap';
+import { SortableElement, SortableHandle } from 'react-sortable-hoc';
+
+import { Tooltip } from '@spinnaker/core';
+
+import { IAmazonPreferredInstanceType } from '../../../../../../instance/awsInstanceType.service';
+import { CostFactor } from '../../../../../../instance/details/CostFactor';
+import { IAmazonInstanceTypeOverride } from '../../../../serverGroupConfiguration.service';
+
+const SortableRow = SortableElement(
```

Comment:
The sortable and non sortable row are very similar. I would prefer to see one component with a prop `sortable: boolean`. 

For the sortable feature you can check out any implementation that uses the `react-sortable-hoc` package.